pkg/square/builder.go

```diff
@@ -22,13 +22,13 @@ type Builder struct {
 	currentSize int
 
 	// here we keep track of the pending data to go in a square
-	txs   [][]byte
-	pfbs  []*coretypes.IndexWrapper
-	blobs []*element
+	Txs   [][]byte
+	Pfbs  []*coretypes.IndexWrapper
+	Blobs []*Element
 
 	// for compact shares we use a counter to track the amount of shares needed
-	txCounter  *shares.CompactShareCounter
-	pfbCounter *shares.CompactShareCounter
+	TxCounter  *shares.CompactShareCounter
+	PfbCounter *shares.CompactShareCounter
```

Comment:
I wanted to try to minimize the repeated code, so by exporting these fields we can reuse a large portion of the builder.

That being said, we can probably do better on this front if we spend more time refactoring